wiki/Tournaments/TWC/2015/de.md

```diff
@@ -2,155 +2,526 @@
-2. Um sich anzumelden, [senden Sie eine PM an Loctav](https://osu.ppy.sh/home/messages/users/71366) mit dem Titel “TWC Registration”.
-   1. Fügen Sie Ihre lokale Zeitzone zum Registrierungs-PM hinzu.
-   2. Sie erhalten eine Bestätigung, dass Ihre Anmeldung erfolgreich angekommen ist. Ihre Anmeldung ist dann in Bearbeitung.
-3. Um gültige und ernsthafte Anmeldungen zu gewährleisten, wird jede Anmeldung und Veränderung vom Turnier Management überprüft.
-   1. Teams werden fürs jeweilige Land zusammengestellt. Jeder Spieler wird als potenzielles Teammitglied in Betracht gezogen. Die Flagge im Profil ist ausschlaggebend.
-   2. Teams werden nach ihren derzeitigen Gameplay Performance und den spezifischen Erfahrungen des jeweiligen Spielers zusammengestellt.
-   3. Teams werden mit Hilfe von Repräsentanten aus der Liste der gültigen Anmeldungen zusammengestellt.
-   4. Bei gleichstarken Spielern wird das Turnier Management möglicherweise das Zufall entscheiden lassen.
+1. Jeder Nutzer, der daran interessiert ist, seinem Landesteam beizutreten, meldet sich individuell an.
+   - Das Turniermanagement wird eine Liste der potentiellen Kandidaten für jedes Länderteam erstellen.
+   - Das Turniermanagement wird einen Kandidaten zum Kapitän des Landesteams ernennen, wenn auch nur temporär.
+   - Der ernannte Kapitän kann sein Landesteam von der Kandidatenliste seines Landes formen.
+2. Um gültige und ernsthafte Anmeldungen zu gewährleisten, wird jede Anmeldung und vom Turniermanagement überprüft.
+   - Jeder registrierte Nutzer wird der Kandidatenliste seines jeweiligen Landes zugeiteilt.  
+   - Um für die Liste akzeptiert zu werden, musst du sicherstellen dass dein globaler Taiko-Performance-Rang über \#8000 liegt.
+   - Um für die Liste akzeptiert zu werden, musst du sicherstellen dass du die [osu!-Communityregeln](/wiki/Rules) in den letzten 12 Monaten nicht gebrochen hast.
+3. Um dich anzumelden, [sendee eine PM an Loctav](https://osu.ppy.sh/home/messages/users/71366) mit dem Titel “TWC Registration”.
+   - Füge deine lokale Zeitzone zum Registrierungs-PM hinzu.
+   - Du erhältst eine Bestätigung, dass deine Anmeldung erfolgreich angekommen ist. Deine Anmeldung ist dann in Bearbeitung.
 4. Die maximale Anzahl an Teams liegt bei 32.
-   1. Die Gesamtanzahl an Teams hängt von den eingegangen Anmeldungen ab.
+   - Die Gesamtanzahl an Teams hängt von den eingegangen Anmeldungen ab.
 5. Alle erfolgreich zusammengestellten Teams werden nach der Registrierungsphase veröffentlicht.
-6. Map Wähler dürfen nicht am Turnier als Spieler teilnehmen.
+6. Beatmap-Wähler dürfen am Turnier nicht als Spieler teilnehmen.
 
 ### Etappenanweisungen
 
-1. In der ersten Etappe (Gruppenphase) werden alle Teams in 8 Gruppen mit jeweils 4 Teams aufgeteilt.
-   1. Dies hängt jedoch an der Gesamtanzahl an Teams ab und kann daher variieren.
+1. In der ersten Etappe (Gruppenetappe) werden alle Teams in 4 Gruppen mit jeweils 6 Teams aufgeteilt.
```

Comment:
I find it weird that this list here has newline characters in-between, but well, that's not of concern now 🤷

Reply:
Just matching the formatting 🤷